vendor/DSCResources/README.md

```diff
@@ -0,0 +1,32 @@
+## DSC Resource Kit Wave 6
+
+This is a fixed version of the [DSC Resource Kit Wave 6](http://gallery.technet.microsoft.com/scriptcenter/DSC-Resource-Kit-All-c449312d)
+
+The fixes are for compatibility with the MOF parser. This is originally from https://github.com/msutter/dsc-resource-kit-wave-6
```

Comment:
I don't understand the commentary... I could codegen everything in the lastest set of resources at https://github.com/PowerShell/DSCResources without making any modification to the MOFs.

Are we noting this temporarily, until we do a mass update of MOF + generated types to catch up to what's presently on GitHub?